src/main/java/javax/jmdns/impl/DNSCache.java

```diff
@@ -52,7 +52,7 @@ public class DNSCache extends ConcurrentHashMap<String, List<DNSEntry>> {
 
     private static final long   serialVersionUID    = 3024739453186759259L;
     
-    private final SimpleLockManager _lm = new SimpleLockManager();
+    private final transient SimpleLockManager _lm = new SimpleLockManager();
```

Comment:
`transistent` is a keyword to change behavoiur of serialization. This field is final so this is thread safe without any further keyword.

Reply:
Final fields are also serialized by default, unless static as far as I know.

Reply:
I did assume this change is for thread safty?

Reply:
Pull request yes. However I crammed in transient at the same time when I realised DNSCache is Serializable.